client/src/components/LoginForm.jsx

```diff
@@ -0,0 +1,109 @@
+import React, { Component } from 'react';
+import validateInput from '../../../server/validators/validatesignin';
+import '../assets/css/signup.css';
+
+class LoginForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
+      password: '',
+      errors: {}
+    };
+    this.onEmailChange = this.onEmailChange.bind(this);
+    this.onPasswordChange = this.onPasswordChange.bind(this);
+    this.onSubmit = this.onSubmit.bind(this);
+  };
+
+  onEmailChange(event) {
```

Comment:
Instead of duplicating methods, why not have one `onChange` method that handles both password and email change?